docs/conf.py

```diff
@@ -11,6 +11,8 @@
 source_suffix = [".rst", ".md"]
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 pygments_style = "sphinx"
+autoclass_content = "both"
+#autodoc_typehints = "description"
```

Comment:
This was for `sphinx.ext.autodoc.typehints` I assume `sphinx_autodoc_typehints` doesn't need this, but not sure why the other project was chosen and it looks like they are [looking for a new maintainer](https://github.com/agronholm/sphinx-autodoc-typehints/issues/146) and it may make sense to go with the main Sphinx project

Reply:
Thanks for the pointer. I've addressed this separately in https://github.com/kislyuk/watchtower/commit/ed0d3cc1613ca5d6a5fd97bd7cb3d30fb5e3de0d.

Reply:
A new maintainer has been found :blush: